app/views/transactions/list.njk

```diff
@@ -45,5 +45,5 @@
   </tbody>
 </table>
 {% else %}
-  <p class="govuk-body" id="no-results">No results match the search criteria.</p>
+  <p class="govuk-body" id="no-results">There are no results for the filters you used.</p>
```

Comment:
Was already like this, but imagine we can probably remove the `id` here too. Fine to merge without changing this though.

Reply:
Will remove it next PR.